core/src/main/scala/stryker4s/config/ConfigReader.scala

```diff
@@ -47,4 +52,15 @@ object ConfigReader extends Logging {
       error(failures.toList.map(_.description).mkString(System.lineSeparator))
       throw ConfigReaderException(failures)
   }
+
+  /**
+    * Sets the logging level to one of the following levels:
+    * OFF, ERROR, WARN, INFO, DEBUG, TRACE, ALL
+    *
+    * @param level the logging level to use
+    */
+  private def setLoggingLevel(level: Level): Unit = {
```

Comment:
It's a bit ugly to have the log level being a side-effect in the ConfigReader, imo. But it's fine for now

Reply:
i know but otherwise we lose all the config logging